recipe/meta.yaml

```diff
@@ -12,6 +12,7 @@ source:
   patches:
     # Fixed: https://github.com/opencv/opencv/blob/master/modules/videoio/src/cap_mjpeg_decoder.cpp#L793
     - cap_mpjpeg_decoder.patch  # [win]
+    - cap_qtkit_100s_fix.patch
```

Comment:
Please add a link to the patch taken from upstream as a comment. Guessing it is this one ( https://github.com/opencv/opencv/pull/6051 ) based on the ref in the issue.